plugins/org.jboss.tools.livereload.core/src/org/jboss/tools/livereload/core/internal/server/jetty/ApplicationsProxyServlet.java

```diff
@@ -61,5 +65,20 @@ protected URI rewriteURI(HttpServletRequest request) {
 		}
 		return null;
 	}
+	
+	/**
+	 * Customize the returned 'location' header to replace the app server port with the proxy port
+	 */
+	@Override
+	protected String filterResponseHeader(HttpServletRequest request, String headerName, String headerValue) {
+		if("Location".equals(headerName)) {
```

Comment:
what if the server actually does  a redirect to different external  server ? should that not be honored ?